pkgs/applications/system/coolercontrol/coolercontrold.nix

```
@@ -37,6 +39,10 @@ rustPlatform.buildRustPackage {
       --replace-fail '/usr/bin' "$out/bin"
   '';
 
+  postFixup = ''
+    addDriverRunpath "$out/bin/$pname"
```

I know this was already like that in the other PR, but JYSK: Please don't rely on `pname` matching the final executable's name. Just hardcode the expected name for now.

```suggestion
    addDriverRunpath "$out/bin/coolercontrold"
```

(Eventually this should prolly use the `mainProgram` attribute from `meta`, but implementing that in an override-compatible manner needs further changes. Adding a note to my todo list, will see when I have the spoons for that…)